oranda.json

```diff
@@ -1,7 +1,10 @@
 {
   "favicon": "https://www.axo.dev/favicon.ico",
   "path_prefix": "oranda",
-  "md_book": "./docs/book/",
+  "mdbook": {
+    "path": "./docs/",
```

Comment:
is the trailing slash here load bearing? if not i feel like it would be best to remove it